### ISSUE TITLE
User search page should indicate if the user is sharing with me

### ISSUE THREAD

**Issue by Flaburgan** [diaspora/diaspora#5796]
As a user, I want to be able to know if the persons displayed in the result list of my search are sharing with me.

We could simply display a green tick next to the aspects dropdown.

**Comment by Globulle** [comment]
I agree. I think the result list should also return all the persons who share with me even if they have disabled "Allow to search me on Diaspora*". Currently the only way to access the profile of some friends of mine is to look for them in my contact list.

**Comment by Flaburgan** [comment]
@Globulle There is an issue about that, but I'm not able to find it.

**Comment by Flaburgan** [comment]
@Globulle I finally found the issue: https://github.com/diaspora/diaspora/issues/4789
